cmd/protoc-gen-gorums/gengorums/gorums.go

```diff
@@ -117,8 +148,24 @@ func hasGorumsType(services []*protogen.Service, gorumsType string) bool {
 	return false
```

Comment:
Replace this `return false` with:
```go
	return hasStrictOrderingType(services, gorumsType)
```
(see also companion comment in gorums_dev.go:14)